packages/react-components/react-list-preview/.npmignore

```diff
@@ -0,0 +1,38 @@
+.storybook/
```

Comment:
AFAIR this file is not needed anymore.

Reply:
yea this was auto generated, will remove